securedrop/models.py

```diff
@@ -288,6 +288,8 @@ def to_json(self):
             'filename': self.filename,
             'size': self.size,
             'journalist_username': self.journalist.username,
+            'journalist_first_name': self.journalist.first_name or '',
```

Comment:
I think these should be `null`/`None` the name isn't set. That seems to more clearly communicate the absence of a value.

Reply:
In Python empty strings are generally preferred to mark unset string values.

Reply:
Yes but this is a JSON API, not just an internal Python-only data representation. When I get data from an API, I would expect a lack of information to have either a missing key or `null` as the value, not `""`. These are very different things.

Reply:
hmm yeah, fair point (I looked at some other JSON APIs, e.g. Github and this is what they are also doing), fixed and repushed